client/src/components/Alert/AlertContainer.tsx

```diff
@@ -1,35 +1,35 @@
-import React from "react";
-import { connect } from "react-redux";
-import { clearAlert } from "../../store/alert/actions";
-import Alert from "./Alert";
+import React, { ReactElement } from 'react';
+import { connect } from 'react-redux';
+import { clearAlert } from '../../store/alert/actions';
+import Alert from './Alert';
 
-interface AlertProps {
-  clearAlert: Function;
-  message: string;
-  color: string;
+interface IAlertProps {
+    message: string;
+    color: string;
+    cleanAlert: any;
 }
 
-interface Alert {
-  message: string;
-  color: string;
+interface IAlert {
+    message: string;
+    color: string;
 }
 
-interface AlertState {
-  alert: Alert;
+interface IAlertState {
+    alert: IAlert;
 }
 
-const AlertContainer: React.FunctionComponent<AlertProps> = ({
+const AlertContainer: React.FunctionComponent<any> = ({
   message,
   color,
-  clearAlert
-}) => <Alert clearAlert={clearAlert} message={message} color={color} />;
+  cleanAlert,
+}: IAlertProps): ReactElement | null => <Alert cleanAlert={cleanAlert} message={message} color={color} />;
```

Comment:
message &&
    <UncontrolledAlert color={color} onClick={(): void => { cleanAlert(); }}>{message}</UncontrolledAlert>

and remove client/src/components/Alert/Alert.tsx

Reply:
fixed